docs/submission/validating-molecular-data.md

```diff
@@ -351,13 +352,16 @@ seq-tools -v
 ```
 To update the client to the latest version, pull the latest version of the repository and run: 
 ```bash
+git pull
 cd seq-tools
 pip3 install .
 ```
 
 </TabItem>
 <TabItem value="docker">
 
+A required pre-requisite is to have Docker  installed and configured on your operating system.  Depending on your permissions, this may or may not be allowed.  In that case, it is best to use the Client version. 
```

Comment:
I think it's just `prerequisite`

Maybe the last line should read
If you do not have permission, it is best to use the Client version.

Reply:
It's just confusing when you say "may or may not be allowed" but then "in that case"... which case?

Reply:
ok changed